src/assets/scripts/client/airport/Arrival/ArrivalBase.js

```diff
@@ -59,6 +59,7 @@ export default class ArrivalBase {
      * @param options {object}
      */
     constructor(airport, options) {
+        console.error('ArrivalBase is a deprecated Class');
```

Comment:
can't wait to get to the deprecation task on this issue.  there are going to be a lot of files going 💥 💥